Formula/sbt.rb

```diff
@@ -21,13 +21,15 @@ def install
 
     (bin/"sbt").write <<~EOS
       #!/bin/sh
-      if [ -f "$HOME/.sbtconfig" ]; then
-        echo "Use of ~/.sbtconfig is deprecated, please migrate global settings to #{etc}/sbtopts" >&2
-        . "$HOME/.sbtconfig"
-      fi
       export JAVA_HOME="${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
       exec "#{libexec}/bin/sbt" "$@"
     EOS
+
+    (bin/"sbtn").write <<~EOS
+      #!/bin/sh
+      export JAVA_HOME="${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
+      exec "#{libexec}/bin/sbtn-x86_64-apple-darwin" "$@"
+    EOS
```

Comment:
Can you apply the same thing above? Without the deprecation this can be done a lot easier
```suggestion
    (bin/"sbtn").write_env_script libexec/"bin/sbtn-x86_64-apple-darwin", Language::Java.overridable_java_home_env
```